spec/acceptance/puppetserver_upgrade_spec.rb

```diff
@@ -17,7 +17,8 @@
     case ENV['BEAKER_PUPPET_COLLECTION']
     when 'puppet7'
       from_version =
-        if fact('os.family') == 'RedHat' && fact('os.release.major') == '9'
+        if (fact('os.family') == 'RedHat' && fact('os.release.major') == '9') ||
+          (fact('os.name') == 'Ubuntu' && fact('os.release.major') == '22.04')
           '7.13.0'
```

Comment:
Like https://github.com/theforeman/puppet-puppet/pull/893#discussion_r1382209344. Perhaps we should skip the whole test if there is just a single version?

Reply:
I was hoping 7.14 just pops up soon and we can use 13->14 as the test.
But yeah, in the meantime we can also skip it totally for EL9 and U22.